experimental/table/shell/src/main/java/alluxio/cli/catalog/command/AttachDatabaseCommand.java

```diff
@@ -61,17 +61,13 @@ public void validateArgs(CommandLine cl) throws InvalidArgumentException {
   }
 
   @Override
-  public int run(CommandLine cl) {
+  public int run(CommandLine cl) throws AlluxioStatusException {
     String[] args = cl.getArgs();
     String dbName = args[0];
     String dbType = args[1];
     Properties p = cl.getOptionProperties(OPTION_OPTION.getOpt());
-    try {
-      mClient.attachDatabase(dbName, dbType, Maps.fromProperties(p));
-    } catch (AlluxioStatusException e) {
-      e.printStackTrace();
-      return 1;
-    }
+    mClient.attachDatabase(dbName, dbType, Maps.fromProperties(p));
+    System.out.println(String.format("Attached database %s (type: %s)", dbName, dbType));
```

Comment:
I feel like this output isn't very helpful since we are just reprinting the user-specified options

Reply:
removed